src/test/java/racing/CarRaceTest.java

```diff
@@ -1,30 +1,95 @@
 package racing;
 
 import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
-import static org.junit.jupiter.api.Assertions.*;
+public class CarRaceTest {
+    final private List<Car> cars = new ArrayList<Car>();
+    final private CarRace carRace = new CarRace();
 
-class CarRaceTest {
+    final private Car car1 = new Car("Tommy", 8);
+    final private Car car2 = new Car("Tammy", 14);
+    final private Car car3 = new Car("Bear", 4);
+    final private Car car4 = new Car("Tonny", 14);
+
+    @BeforeEach
+    void init() {
+        cars.add(car1);
+        cars.add(car2);
+        cars.add(car3);
+        cars.add(car4);
+    }
 
     @Test
-    void getWinners() {
-        Car car1 = new Car("Tom", 8);
-        Car car2 = new Car("Popo", 14);
-        Car car3 = new Car("Papa", 14);
+    @DisplayName("자동차 이름을 입력 받는다.")
+    void enterCarName() {
+        String entry = "Tommy,bear,Bear,tommy,ToMmy";
+
+        List<String> carEntry = Stream.of(entry.split(",")).collect(Collectors.toList());
+
+        Assertions.assertThat(carEntry.get(0)).isEqualTo("Tommy");
+        Assertions.assertThat(carEntry.get(1)).isEqualTo("bear");
+        Assertions.assertThat(carEntry.get(2)).isEqualTo("Bear");
+        Assertions.assertThat(carEntry.get(3)).isEqualTo("tommy");
+        Assertions.assertThat(carEntry.get(4)).isEqualTo("ToMmy");
+        Assertions.assertThatExceptionOfType(IndexOutOfBoundsException.class)
+                .isThrownBy(() -> {
+                    carEntry.get(5);
+                });
+    }
```

Comment:
의미 있는 테스트인가요? 어떤걸 테스트하는 걸까요? 단순히 split이 잘 되었는지에 대한 테스트인가요?

```suggestion
    @DisplayName("자동차 이름을 입력 받는다.")
    void enterCarName() {
        String entry = "Tommy,bear,Bear,tommy,ToMmy";

        List<String> carEntry = Stream.of(entry.split(",")).collect(Collectors.toList());

        assertThat(carEntry.get(0)).isEqualTo("Tommy");
        assertThat(carEntry.get(1)).isEqualTo("bear");
        assertThat(carEntry.get(2)).isEqualTo("Bear");
        assertThat(carEntry.get(3)).isEqualTo("tommy");
        assertThat(carEntry.get(4)).isEqualTo("ToMmy");
        assertThatExceptionOfType(IndexOutOfBoundsException.class)
                .isThrownBy(() -> {
                    carEntry.get(5);
                });
    }
```

static import를 하는 것이 어떨까요?

Reply:
해당 테스트는 스플릿이 제대로 된 정도의 테스트를 의도한것 입니다.